ProvinceMapper/Source/Frames/Images/ImageCanvas.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+#include <wx/wxprec.h>
+#ifndef WX_PRECOMP
+#include <wx/wx.h>
+#endif
+
+#include "../../Provinces/Province.h"
+
+enum class ImageTabSelector
+{
+	SOURCE,
+	TARGET
+};
+
+class LinkMappingVersion;
+class ImageCanvas: public wxScrolledCanvas
+{
+  public:
+	ImageCanvas(wxWindow* parent, ImageTabSelector theSelector, const std::shared_ptr<LinkMappingVersion>& theActiveVersion, wxImage* theImage);
+
+	[[nodiscard]] auto getHeight() const { return height; }
+	[[nodiscard]] auto getWidth() const { return width; }
+	[[nodiscard]] const auto& getImageData() const { return imageData; }
+
+	void clearBlackList() { blackList.clear(); }
+	void generateBlackList();
+	void applyBlackList();
+	void restoreImageData();
+	
+  private:
+	wxImage* image;
+	unsigned char* imageData;
+	size_t imageDataSize = 0;
+	int height = 0;
+	int width = 0;
+	std::vector<Pixel> blackList;
```

Comment:
Too easy to confuse with the word blacklist, which has a different meaning. Perhaps linkedPixels or some such?

Update function names as well.